src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,17 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        if (numbers.size() == 0) {
+            throw new RuntimeException("Can't get min value from list: " + numbers);
+        }
+        return numbers.stream()
+                .map(separation -> separation.split(","))
+                .flatMap(transition -> Arrays.stream(transition))
```

Comment:
```suggestion
                .flatMap(nums -> Arrays.stream(nums.split(",")))
```
we can combine these 2 operations into one